elements/userlevel/todpdkdevice.hh

```diff
@@ -0,0 +1,174 @@
+#ifndef CLICK_TODPDKDEVICE_USERLEVEL_HH
+#define CLICK_TODPDKDEVICE_USERLEVEL_HH
+
+#include <click/element.hh>
+#include <click/sync.hh>
+#include <click/timer.hh>
+#include <click/dpdkdevice.hh>
+
+CLICK_DECLS
+
+/*
+=title ToDpdkDevice
+
+=c
+
+ToDpdkDevice(PORT [, QUEUE, IQUEUE, BLOCKING, BURST, TIMEOUT, NDESC]])
```

Comment:
Oops! The configuration argument order defined in the source isn't in sync (you have BLOCKING, IQUEUE in the source). Suggest you make these keyword arguments only; the documentation would be updated with `I<keywords>` and the `read_p`s would change to `read`.